internal/impl/tmpl/direnvrc.tmpl

```diff
@@ -0,0 +1,5 @@
+use_devbox() {
```

Comment:
is direnvrc  a standard name? It looks like direnv uses it for custom extensions. Is this an extension?